site/_posts/2019-07-26-Announcing-Icanchuk-Beta1.html.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Announcing Ivanchuk Beta1 Release
+author: jprause
+date: 2019-07-26 07:49:24 UTC
+comments: true
+published: true
+tags: releases announcements
+---
+
+On behalf of the ManageIQ team, we're pleased to announce the Beta1 release for ManageIQ Ivanchuk! You can download the Ivanchuk Beta1 Release [here](http://manageiq.org/download/)
+
+Ivanchuk is the next milestone release for ManageIQ hybrid IT management platform. With each release, ManageIQ gets more robust and feature complete across providers. Here are just a few of the new or updated features.
+
+* Add option to schedule automation requests via workflow
+* Add a tool to assist in reconnecting vms
+* Add zone to service provisioning
+* Add method to copy service templates
+* Allow user to select Network Port for provisioning
+* Add support for exporting and importing reports
+
+Take a look at the [changelog](https://github.com/ManageIQ/manageiq/blob/ivanchuk/CHANGELOG.md/) for more details.
+
+We encourage you to [download](http://manageiq.org/download/) the Hammer Beta Release and try it out!
```

Comment:
s/Hammer/Ivanchuk/ :wink:

Reply:
Thanks @cybette I had missed one prior, caught it, and thought, "there couldn't possibly be any others".

Reply:
been there, done that :P